Calibration/LumiAlCaRecoProducers/plugins/RawPCCProducer.cc

```diff
@@ -95,37 +97,36 @@ void RawPCCProducer::globalEndLuminosityBlockProduce(edm::LuminosityBlock& lumiS
   //new vector containing clusters per bxid with afterglow corrections
   std::vector<float> corrClustersPerBXOutput(LumiConstants::numBX, 0);
 
-  //The indicies of all the good modules - not vetoed
-  std::vector<int> goodMods;
-
+  //////////////////////////////////
+  /// read input , clusters per module per bx
+  /////////////////////////////////
   edm::Handle<reco::PixelClusterCounts> pccHandle;
   lumiSeg.getByToken(pccToken_, pccHandle);
-
   const reco::PixelClusterCounts& inputPcc = *(pccHandle.product());
+  auto modID = inputPcc.readModID();                //vector with Module IDs 1-1 map to bunch x-ing in clusers
+  auto events = inputPcc.readEvents();              //vector with total events at each bxid.
+  auto clustersPerBXInput = inputPcc.readCounts();  //cluster counts per module per bx
 
-  //vector with Module IDs 1-1 map to bunch x-ing in clusers
-  auto modID = inputPcc.readModID();
-  //vector with total events at each bxid.
-  auto events = inputPcc.readEvents();
-  auto clustersPerBXInput = inputPcc.readCounts();
-
-  //making list of modules to sum over
+  ////////////////////////////
+  ///Apply the module veto
+  ///////////////////////////
+  std::vector<int> goodMods;
   for (unsigned int i = 0; i < modID.size(); i++) {
     if (std::find(modVeto_.begin(), modVeto_.end(), modID.at(i)) == modVeto_.end()) {
       goodMods.push_back(i);
     }
   }
-
-  //summing over good modules only
   for (int bx = 0; bx < int(LumiConstants::numBX); bx++) {
     for (unsigned int i = 0; i < goodMods.size(); i++) {
       clustersPerBXOutput.at(bx) += clustersPerBXInput.at(goodMods.at(i) * int(LumiConstants::numBX) + bx);
     }
   }
 
+  //////////////////////////////
+  //// Apply afterglow corrections
+  //////////////////////////////
   std::vector<float> correctionScaleFactors;
   if (applyCorr_) {
-    //Get PCC corrections from the event setup through a token
     const auto pccCorrections = &iSetup.getData(lumiCorrectionsToken_);
```

Comment:
Why was the comment above "//Get PCC corrections from the event setup through a token" deleted? I think that was a nice explanation on what happens here

Reply:
the comment didn't provide any additional info.